browser/src/slideshow/LayerDrawing.ts

```diff
@@ -761,6 +805,47 @@ class LayerDrawing {
 			oldCallback.call(this.helper);
 	}
 
+	private prefetchNextCompressedSlide(): void {
+		if (this.compressedPrefetchSlideHash) {
+			const slideInfo = this.getSlideInfo(this.compressedPrefetchSlideHash);
+			this.compressedPrefetchSlideHash = null;
+
+			if (slideInfo?.next) {
+				this.prefetchCompressedSlide(slideInfo.next);
+			}
+		}
+	}
+
+	private prefetchCompressedSlide(slideHash: string) {
+		const slideInfo = this.getSlideInfo(slideHash);
+		if (!slideInfo) {
+			console.warn(`SlideInfo not found for hash: ${slideHash}`);
+			return;
+		}
+
+		if (this.isSlideAlreadyCached(slideHash)) {
+			this.prefetchNextSlide(slideInfo);
+			return;
+		}
+		this.compressedPrefetchSlideHash = slideHash;
+		setTimeout(() => {
```

Comment:
can we setup some deterministic/synchronized way of calling that?

Reply:
or why 1s ?
doesn't it depend on any other "task" ?

Reply:
It doesn't matter possibly if it's `1s` or `0`. The question is: `does it require any other thing to be done to work properly?` or any other thing has to wait for it? do we need to cancel it in some cases?

if: no - we can live with it possibly (as it is just a prefetching timeout)